lib/third_party/mcu_vendor/espressif/esp-idf/README.md

```diff
@@ -96,7 +104,7 @@ The simplest way to use the partition table is to `make menuconfig` and choose o
 
 In both cases the factory app is flashed at offset 0x10000. If you `make partition_table` then it will print a summary of the partition table.
 
-For more details about partition tables and how to create custom variations, view the [`docs/api-guides/partition-tables.rst`](docs/api-guides/partition-tables.rst) file.
+For more details about partition tables and how to create custom variations, view the [`docs/en/api-guides/partition-tables.rst`](docs/en/api-guides/partition-tables.rst) file.
```

Comment:
I just did a spot check of a couple of links, but this link is not working for me.

Reply:
There is no `docs` directory here, since getting started documentation provides references to https://docs.espressif.com/projects/esp-idf/en/latest/get-started/index.html